frontend/public/components/dashboard/dashboards-page/cluster-dashboard/details-card.tsx

```diff
@@ -136,6 +136,8 @@ export const DetailsCard = withDashboardResources(
     const infrastructurePlatform = getInfrastructurePlatform(infrastructure);
     const infrastuctureApiUrl = getInfrastructureAPIURL(infrastructure);
 
+    const serviceLevelTitle = useServiceLevelTitle();
```

Comment:
Do you mind moving this a little up just below the last useEffect hook so all the hooks are together at the top?